examples/cache-with-aws/README.md

```diff
@@ -0,0 +1,30 @@
+# Java Client SDK
+
+_Read this in other languages_: [日本語](README.ja.md)
+
+<br>
+
+## Running the Examples
+
+- You do not need gradle to be installed
+- JDK 14 or above is required to run the example
+- A Momento Auth Token is required, you can generate one using the [Momento CLI](https://github.com/momentohq/momento-cli)
+
+### Basic
```

Comment:
Can you add a sentence or two here explaining what this demo does? basically, say it's the same as the `cache` demo but gets a token from Secrets Manager. Also would include some info letting them know that they need to store their token in secrets manager first, and link to Kirk's Secrets Manager doc on the dev docs site.